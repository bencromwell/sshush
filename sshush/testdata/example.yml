---
global:
  UseRoaming: "no"

default:
  User: ben
  IdentityFile: ~/.ssh/id_rsa

web_servers:
  Prefix: projects-
  Config:
    Port: 2201
    IdentityFile: ~/.ssh/digital_ocean
  Hosts:
    do-1:
      HostName: projects-do-1.example.com
    do-2:
      HostName: projects-do-2.example.com
    aws:
      HostName: projects-aws.example.com
      IdentityFile: ~/.ssh/aws

raspberry_pis:
  Config:
    User: pi
  Hosts:
    pi1:
      HostName: 192.168.0.107
    pi2:
      HostName: 192.168.0.108
      LocalForward:
        - 8080 127.0.0.1:80
        - 8443 127.0.0.1:443

list_config_test_case:
  Config:
    LocalForward:
      - 8080 127.0.0.1:80
      - 8443 127.0.0.1:443
  Hosts:
    lf_test_1: 192.168.0.109

local:
  Hosts:
    router:
      HostName: 192.168.0.1
      User: root
    kodi:
      HostName: 192.168.0.200

work:
  Config:
    User: bcromwell
  Hosts:
    workpc:
      HostName: 10.0.0.80
    gitlab:
      HostName: 10.0.0.30
    jenkins:
      HostName: 10.0.0.20
