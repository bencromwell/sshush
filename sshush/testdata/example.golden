# Generated by sshush v0.0.0-dev
# From testdata/example.yml

# web_servers
Host projects-aws
    HostName projects-aws.example.com
    IdentityFile ~/.ssh/aws
    Port 2201
    User ben

Host projects-do-1
    HostName projects-do-1.example.com
    IdentityFile ~/.ssh/digital_ocean
    Port 2201
    User ben

Host projects-do-2
    HostName projects-do-2.example.com
    IdentityFile ~/.ssh/digital_ocean
    Port 2201
    User ben

# raspberry_pis
Host pi1
    HostName 192.168.0.107
    IdentityFile ~/.ssh/id_rsa
    User pi

Host pi2
    HostName 192.168.0.108
    IdentityFile ~/.ssh/id_rsa
    LocalForward 8080 127.0.0.1:80
    LocalForward 8443 127.0.0.1:443
    User pi

# list_config_test_case
Host lf_test_1
    HostName 192.168.0.109
    IdentityFile ~/.ssh/id_rsa
    LocalForward 8080 127.0.0.1:80
    LocalForward 8443 127.0.0.1:443
    User ben

# local
Host kodi
    HostName 192.168.0.200
    IdentityFile ~/.ssh/id_rsa
    User ben

Host router
    HostName 192.168.0.1
    IdentityFile ~/.ssh/id_rsa
    User root

# work
Host gitlab
    HostName 10.0.0.30
    IdentityFile ~/.ssh/id_rsa
    User bcromwell

Host jenkins
    HostName 10.0.0.20
    IdentityFile ~/.ssh/id_rsa
    User bcromwell

Host workpc
    HostName 10.0.0.80
    IdentityFile ~/.ssh/id_rsa
    User bcromwell

# Global config
Host *
    UseRoaming no
